src/Common/Admin/Entities/Paragraph.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ *
+ */
+
+declare( strict_types=1 );
+
+namespace TEC\Common\Admin\Entities;
+
+/**
+ * Class Paragraph
+ *
+ * @since TBD
+ */
+class Paragraph implements Element {
+
+	/**
+	 * The paragraph content.
+	 *
+	 * @var string
+	 */
+	private string $content = '';
+
+	/**
+	 * @var string[]
+	 */
+	private array $classes;
+
+	/**
+	 * Paragraph constructor.
+	 *
+	 * @param string $content The content for the paragraph.
+	 */
+	public function __construct( string $content, $classes = [] ) {
+		$this->content = $content;
+		$this->classes = array_filter(
+			$classes,
+			function( $class ) {
+				return is_string( $class );
+			}
+		);
+	}
+
+	/**
+	 * Render the paragraph content.
+	 *
+	 * @return void
+	 */
+	public function render() {
+		printf(
+			'<p class="%s">%s</p>',
+			esc_attr( implode( ' ', $this->classes ) ),
```

Comment:
tribe_classes() will render the attribute from an array (including the class="") and do your escaping: 
https://github.com/the-events-calendar/tribe-common/blob/master/src/functions/template-tags/html.php#L31